bazel/repositories.bzl

```diff
@@ -211,7 +211,16 @@ def _go_deps(skip_targets):
     # Keep the skip_targets check around until Istio Proxy has stopped using
     # it to exclude the Go rules.
     if "io_bazel_rules_go" not in skip_targets:
+        _repository_impl(
+            name = "com_github_golang_protobuf",
+            patches = [
+                "@io_bazel_rules_go//third_party:com_github_golang_protobuf-gazelle.patch",
```

Comment:
Woohoo! Can you add a comment on why these are needed and when we can get rid of the patches?